examples/step-68/doc/intro.dox

```diff
@@ -0,0 +1,234 @@
+timestep to ensure particles will at most move into the ghost layer of the
+local process and can therefore be send to neighboring processes automatically.
+
+<h4>Balancing mesh and particle load</h4>
+
+The last challenge that arises in parallel distributed computations using
+particles is to balance the computational load between work that is done on the
+grid, for example solving the finite-element problem, and the work that is done
+on the particles, for example advecting the particles or computing the forces
+between particles or between particles and grid. By default, for example in
+step-40, deal.II distributes the background mesh as evenly as possible between
+the available processes, that is it balances the number of cells on each
+process. However, if some cells own many more particles than other cells, or if
+the particles of one cell are much more computationally expensive than the
+particles in other cells, then this problem no longer scales efficiently (for a
+discussion of what we consider "scalable" programs, see @ref
+GlossParallelScaling "this glossary entry"). Thus, we have to apply a form of
+"load balancing", which means we estimate the computational load that is
+associated with each cell and its particles. Repartitioning the mesh then
+accounts for this combined computational load instead of the simplified
+assumption of the number of cells.
+
+In this section we only discussed the particle-specific challenges in distributed
+computation. Parallel challenges that particles share with
+finite-element solutions (parallel output, data transfer during mesh
+refinement) can be addressed with the solutions found for
+finite-element problems already discussed in other examples.
+
+<h3>The testcase</h3>
+
+In the present step, we use particles as massless tracers to illustrate
+the dynamics of a particular vortical flow : the Rayleigh-Kothe Vortex. This flow pattern
+is generally used as a complex test case for interface tracking methods
+(e.g. volume-of-fluid and level set approches) since
+it leads to strong rotation and elongation of the fluid (Blais et al. 2013).
+
+The stream function $\Psi$ of this Rayleigh-Kothe vortex is defined as:
+
+@f[
+\Psi = \frac{1}{\pi} sin^2 (\pi x) \sin^2 (\pi y) \cos \left( \pi \frac{t}{T} \right)
+@f]
+where $T$ is the period of the flow. The velocity profile in 2D ($\textbf{u}=[u,v]^T$) is :
+@f{eqnarray*}
+   u &=&  - \frac{\partial\Psi}{\partial y} = -2 \sin^2 (\pi x) \sin (\pi y) \cos (\pi y)  \cos \left( \pi \frac{t}{T} \right)\\
+   v &=&  \frac{\partial\Psi}{\partial x} = 2 \cos(\pi x) \sin(\pi x) \sin^2 (\pi y) \cos \left( \pi \frac{t}{T} \right)
+@f}
+
+It can be seen that this velocity reverses periodically due to the term
+$\cos \left( \pi \frac{t}{T} \right)$ and that material will end up at its
+starting position after every period of length $t=2T$. We will run this tutorial
+program for exactly one period and compare the final particle location to the
```

Comment:
I would have expected that you end back where you were at `t=T`. This would also avoid the ambiguity in the second sentence: is "one period" equal to T or 2T? In line 189 above, you seem to suggest that a period is T, but stuff only ends up where it started after 2T...

Reply:
Yeah that was confusing. I kept the things the same, just mentioned that T is the half period instead. I prefer to keep it that way since this is generally how we write this flow profile in the literature.